<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>851</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QStackedWidget" name="mainWindowstackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="ScreenLogin" name="pageLogin"/>
      <widget class="ScreenRegister" name="pageRegister"/>
      <widget class="ScreenMainWork" name="pageWork"/>
      <widget class="ScreenUserProfile" name="pageProfile"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1199</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="separator"/>
    <addaction name="userProfileAction"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="userProfileAction">
   <property name="text">
    <string>Профиль пользователя</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScreenLogin</class>
   <extends>QWidget</extends>
   <header>screen_login.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScreenRegister</class>
   <extends>QWidget</extends>
   <header>screen_register.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScreenMainWork</class>
   <extends>QWidget</extends>
   <header>screen_main_work.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScreenUserProfile</class>
   <extends>QWidget</extends>
   <header>screen_user_profile.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
